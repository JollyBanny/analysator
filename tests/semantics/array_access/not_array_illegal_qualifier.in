var arr1: integer;

begin
    write(arr1[1]);
end.