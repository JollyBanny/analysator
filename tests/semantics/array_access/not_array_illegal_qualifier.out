[Line: 4; Column: 16]	illegal qualifier
